src/index.js

```diff
@@ -8,7 +10,7 @@ import isConstructor from './helpers/isConstrutor';
 
 const cwd = process.cwd();
 
-const mapRoutes = (routes, pathToController) => {
+const mapRoutes = (routes, pathToController, middlewareGeneral) => {
```

Comment:
`middlewareGeneral` could be optional, i.e. it could be `middlewareGeneral = []` with an empty array as default.

Reply:
By doing so, more than one middleware can be added generally.